pubsublite/streaming-analytics/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Pub/Sub Lite to Clous Storage",
+  "description": "An Apache Beam streaming pipeline that reads messages from Pub/Sub Lite, applies fixed windowing on the messages, and writes the results to files on Cloud Storage",
+  "parameters": [
+    {
+      "name": "subscription",
+      "label": "Pub/Sub Lite subscription.",
+      "helpText": "Pub/Sub subscription to read from."
```

Comment:
Can we specify the format (i.e. just the name or the full subscription path?) of the subscription here? Having a validation regex would also be nice.

Reply:
Done.